newsletter/_posts/2016-05-17-workshop-lal.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Second HSF Workshop
+author: Michel Jouvin
+layout: newsletter
+---
+
+# Second HSF Workshop (May 2-4, 2016)
+
+The HSF had its second workshop at LAL (Orsay, France) on May 2-4, 2016. It was well attended (70 people) representing many different laboratories, universities and institutions (~30) from both sides of the Atlantic, involved in different experiments (LHC, BELLE II, Intensity Frontier...). We were happy to see a lot of new faces! [Agenda](https://indico.cern.ch/event/496146/timetable/) was a mix of (lively!) general discussions and topical/hands-on sessions. Live notes of these discussions are [available](????). This newsletter summarizes them.
```

Comment:
I would change the numbering there. We had the kick-off one at CERN, the first follow up at SLAC, the second follow up at LAL.